server/tso/allocator_manager.go

```diff
@@ -245,6 +245,25 @@ func (am *AllocatorManager) GetDCLocationInfo(dcLocation string) (DCLocationInfo
 	return infoPtr.clone(), true
 }
 
+// CleanUpDCLocation cleans up certain server's DCLocationInfo
+func (am *AllocatorManager) CleanUpDCLocation() error {
+	serverID := am.member.ID()
+	dcLocationKey := am.member.GetDCLocationPath(serverID)
+	// remove etcd
+	if resp, err := kv.
+		NewSlowLogTxn(am.member.Client()).
+		Then(clientv3.OpDelete(dcLocationKey)).
+		Commit(); err != nil {
+		return errs.ErrEtcdTxnInternal.Wrap(err).GenWithStackByCause()
+	} else if !resp.Succeeded {
+		return errs.ErrEtcdTxnConflict.FastGenByArgs()
+	}
+	log.Info("delete the dc-location key previously written in etcd",
```

Comment:
When there is no dc-location key existing in etcd, we don't need to print this log?

Reply:
Do you think it's better to print the exact information after calling function GetClusterDCLocationsFromEtcd to make sure it doesn't exist, or to print an ambiguous log like "try to delete"?